### ISSUE TITLE
Add data renderer for CFString types

### ISSUE THREAD

**Issue by jonpalmisc** [Vector35/workflow_objc#2]
Rendering CFString instances in linear view as Objective-C style strings, e.g. `@"Hello, world!"`, would be far more useful than seeing the structure itself.

The code for this is already done, just needs to be cleaned up, checked-in, and merged.

**Comment by cxnder** [comment]
Resolved with #46 / 3720bcbf6c967b06d8cdf51701dc2cf88cb0499d

**Comment by jonpalmisc** [comment]
This issue refers to rendering CFString structures in linear view (not references in HLIL, etc.). From reading the commit I don't believe this resolves this.

**Comment by psifertex** [comment]
Isn't that exactly what a datarenderer does? 

https://github.com/Vector35/workflow_objc/pull/50/files

**Comment by jonpalmisc** [comment]
Yes, if considering what this original issue was made for. Kat's changes (fdc174dbd2245cb8d95c7c1c2d89898168312e33) render CFStrings as string literals inside of HLIL, i.e. a "code renderer" which we don't support yet.

The pull request you linked is separate (but addresses the original issue and the reason I reopened this).
